{{cookiecutter.repo_name}}/.coveragerc

```diff
@@ -1,2 +1,2 @@
 [report]
-omit = */migrations/*,{{ cookiecutter.project_name }}/**,manage.py,
+omit = */migrations/*,{{ cookiecutter.project_name }}/**,manage.py,{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}
```

Comment:
The first one  (`{{ cookiecutter.project_name }}/`) will exclude _everything_ so that has to be removed.